tests/unit/processing_container/test_process.py

```diff
@@ -1,38 +1,37 @@
-import pathlib
-import requests
+import os
 import shutil
-import tempfile
 from unittest.mock import patch
 
+import requests
+
 from backend.corpora.common.corpora_orm import (
     CollectionVisibility,
 )
 from backend.corpora.dataset_processing import process
-from backend.corpora.dataset_processing.process import make_cxg, make_seurat, make_loom
 from tests.unit.backend.fixtures.mock_aws_test_case import CorporaTestCaseUsingMockAWS
 
 
 class TestDatasetProcessing(CorporaTestCaseUsingMockAWS):
+    """
+    This test class is intended to exercise the upload pipeline processing, running within
+    a "corpora-upload" (upload processing) Docker container. As the Docker container is configured
+    with all necessary software dependencies, manually running this test outside of Docker
+    (e.g. within an IDE when making changes to the test itself) will require you to:
```

Comment:
Yeah, it was a bit of a minefield figuring out how to actually do development on the tests, since I wanted to use my IDE for test debugging.  Probably can set this up with some Docker config, but that seemed like a rabbit hole.